src/config.c

```diff
@@ -144,6 +144,7 @@ configEnum protected_action_enum[] = {
 configEnum cluster_preferred_endpoint_type_enum[] = {
     {"ip", CLUSTER_ENDPOINT_TYPE_IP},
     {"hostname", CLUSTER_ENDPOINT_TYPE_HOSTNAME},
+    {"nodename", CLUSTER_ENDPOINT_TYPE_NODENAME},
```

Comment:
I wasn't thinking of this as a different endpoint type, just a way to identify a node in a cluster logically. The use case I had in mind was being able to assign an EC2 ID to a node to be able to look it up separately.

Reply:
Yeah.